src/World.cpp

```diff
@@ -3051,7 +3051,11 @@ void cWorld::QueueTask(cTaskPtr a_Task)
 }
 
 
-
+void cWorld::ScheduleTask(int a_DelayTicks, std::function<void (cWorld&)> a_Func)
+{
+	cTaskLambda task = cTaskLambda(a_Func);
```

Comment:
Ah ha! Makes sense. There's my C# background shining through.
Thanks for the help!